tests-e2e/youtrack/expected-errors.txt

```diff
@@ -0,0 +1 @@
+src/tickets.php:3:Should have been resolved in WI-1: fix me.
```

Comment:
this test is missing the github action part, therefore did not ran:

https://github.com/staabm/phpstan-todo-by/pull/71/files#diff-194218c48b9a0cdd03974145733804c2d992ca818529fe2fa69a501d8b5b1cc3R29-R34

Reply:
Added.